src/fec_rs_fnt.h

```diff
@@ -79,16 +78,16 @@ class RsFnt : public FecCode<T> {
         // with this encoder we cannot exactly satisfy users request, we need to
         // pad n = minimal divisor of (q-1) that is at least (n_parities +
         // n_data)
-        n = this->gf->get_code_len_high_compo(n_parities + n_data);
+        this->n = this->gf->get_code_len_high_compo(n_parities + n_data);
 
         // compute root of order n-1 such as r^(n-1) mod q == 1
-        this->r = this->gf->get_nth_root(n);
+        this->r = this->gf->get_nth_root(this->n);
 
         // std::cerr << "n=" << n << "\n";
         // std::cerr << "r=" << r << "\n";
 
         int m = arith::get_smallest_power_of_2<int>(n_data);
-        this->fft = new fft::Radix2<T>(this->gf, n, m, pkt_size);
+        this->fft = new fft::Radix2<T>(this->gf, this->n, m, pkt_size);
```

Comment:
Not directly related to this patch, but I just noticed that we didn't document the param `n` in Doxygen for the `fft::Radix2` in `fft_2n.h`.